custom_components/bluetti_bt/coordinator.py

```diff
@@ -180,7 +184,7 @@ async def _async_update_data(self):
         parsed_data: dict = {}
 
         try:
-            async with async_timeout.timeout(15):
+            async with async_timeout.timeout(45):
```

Comment:
Maybe 45s is excessive, but max packs is 6 so just added 30s.  For persistent connection, it's really quick, but not sure how long it takes with transient connection.